app/src/main/java/org/scesi/cappuchino/ui/utils/SearchBar.kt

```diff
@@ -0,0 +1,158 @@
+package org.scesi.cappuchino.ui.utils
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.scesi.cappuchino.R
+
+sealed class SearchCategory(
+    var searchCriteria: String
+) {
+    data class Subject(
+        val name: String
+    ) : SearchCategory(name)
+
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SearchBar(career: List<SearchCategory>, onClick: () -> Unit) {
+
+    var searchText by remember { mutableStateOf("") }
+    val filteredCarreras = career.filter { carrera ->
+        carrera.searchCriteria.contains(searchText, ignoreCase = true)
+    }
+
+    OutlinedTextField(
```

Comment:
this is getting overlapped with the careers section because is not enclosed with a column, do it with a column, and the overlap will be gone

Reply:
I don't really sure that this will work, but I did